cli/src/config/templates.toml

```diff
@@ -90,8 +90,9 @@ if(root,
     ),
   )
 )
+
 '''
-builtin_log_compact = '''
+'builtin_log_compact_desc(full_description)' = '''
```

Comment:
nit: maybe better to extract the first line (= `separate(" ", .. if(conflict ..))`) as `format_compact_commit_header(commit)`? I don't know if we want to reuse this function for other templates, but user might want to choose where to put "(empty)" marker.